dnsdbq.c

```diff
@@ -95,12 +94,14 @@ const struct verb verbs[] = {
 	{ "lookup", "/lookup", lookup_ok,
 	  present_text_lookup,
 	  present_json_lookup,
-	  present_csv_lookup },
+	  present_csv_lookup,
+	  present_minimal_lookup },
 	{ "summarize", "/summarize", summarize_ok,
 	  present_text_summarize,
 	  present_json_summarize,
-	  present_csv_summarize },
-	{ NULL, NULL, NULL, NULL, NULL, NULL }
+	  present_csv_summarize,
+	  NULL },
```

Comment:
need to handle missing a present_minimal_XX method:
`$ ./dnsdbq -p minimal -r \*.uber.com -l0 -B6m -V summarize`
`Segmentation fault: 11`

Reply:
fixed shortly.